.rubocop.yml

```diff
@@ -407,7 +406,6 @@ Style/Copyright:
 
 Style/DocumentationMethod:
   RequireForNonPublicMethods: false
->>>>>>> master
```

Comment:
You found it! Looks like you had a minor merge conflict that was borking the hound.